apps/editing-toolkit/editing-toolkit-plugin/whats-new/menu/index.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * WordPress dependencies
+ */
+import { registerPlugin } from '@wordpress/plugins';
+import { ToolsMoreMenuGroup } from '@wordpress/edit-post';
```

Comment:
I think the reason this doesn't work is that `ToolsMoreMenuGroup` is not exported from `@wordpress/edit-post`.

Instead, we need to reference the Fill manually. We need to import the `Fill` component from `@wordpress/components`

```suggestion
import { Fill } from '@wordpress/components';
```